encryption_works.md

```diff
@@ -411,6 +422,8 @@ From here, follow the advice on the [Tails installation page](https://tails.boum
 
 ![Verifying checksums for the Tails download.](images/tails_verification.png)
 
+Here, we can see that the *primary* and *subkey fingerprints* are the same for both the Tails website and my command line, which means the software I downloaded was what the developers intended me to get. 
```

Comment:
Not quite. The primary vs. subkey fingerprints are a technical detail that's not really important here. They have nothing to do with whether the software successfully verified. The indication that it verified is that it says "Good signature" instead of "Bad signature".

Note that in this example, you haven't verified the Tails developers key at all, so it's only as secure as the Tails website/HTTPS/etc. (same caveats as my earlier comment about verifying keys obtained from the Internet).

While you don't have to go into great length about how to verify the Tails signing key in this section, you might want to mention those caveats. Also, you could link to the [Tails documentation on verifying their release signing key](https://tails.boum.org/doc/get/trusting_tails_signing_key/index.en.html).